pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "knockoff"
-version = "3.1.1"
+version = "4.0.0"
 description = "Library for generating and bootstrapping mock data"
 authors = ["Gregory Yu <gregory.yu@nike.com>"]
 readme = "README.md"
```

Comment:
Can we add `maintainers` section in this PR?

```
maintainers = [
"Mark Niedzielski <Mark.Niedzielski@nike.com>", 
"Mohamed Abdul Huq Ismail <Abdul.Ismail@nike.com>"
]
```

Reply:
done